### ISSUE TITLE
Broker instance calculation is creating more brokers than needed

### ISSUE THREAD

**Issue by lulf** [EnMasseProject/enmasse#2042]
**Describe the bug**
When creating 3900 addresses with broker credit 0.001, one ends up with 6 brokers created instead of the needed 4.

The following log from the standard-controller highlight the issue:

```
{broker={broker-pooled-acg3p61m80-1={used=1.0040000000000002, needed=2}, broker-pooled-acg3p61m80-2={used=1.0119999999999993, needed=2}, broker-pooled-acg3p61m80-0={used=0.9990000000000008, needed=1}, broker-pooled-acg3p61m80-3={used=0.8850000000000007, needed=1}}}
```

Here, there are broker pods that have a usage > 1 which should not be allowed to begin with.

**Comment by lulf** [comment]
I think this is related to the stale addresses:

First the usage is calculated for addresses not in Pending state, then usage is added for the ones that are moved from Pending to the Configuring state. Second iteration, usage is calculated based on addresses in Configuring state. However, some of them are stale (aka. still in Pending state), so they are not taken into account when calculating the usage. Still, new Pending addresses moving to Configured are added to the usage. This way, usage can exceed 1

CC @k-wall

**Comment by lulf** [comment]
Fixed in #2101
